src/SWP/Bundle/MultiTenancyBundle/spec/DependencyInjection/Compiler/RegisterOrganizationFactoryPassSpec.php

```diff
@@ -0,0 +1,111 @@
+<?php
```

Comment:
I honestly cannot see the point of these spec tests. The only time any one of them has failed is when I've purposefully changed something. I can maybe see the point in writing them first, so you define the behaviour before actually coding it, but if there is a reason to write them afterwards, I'd love to know what it is.